core/app/assets/javascripts/refinery/core.js.erb

```diff
@@ -1,11 +1,12 @@
 $(document).ready(function(){
   // focus first field in an admin form.
   $('form input[type=text]:first').focus();
-
-  init_flash_messages();
```

Comment:
So it'll only be run inside layouts including `admin.js` ? Doesn't that mean _not_ the Preview controller?